docker/configure_workers_and_start.py

```diff
@@ -496,6 +482,10 @@ def generate_worker_files(environ, config_path: str, data_dir: str):
 
     # Finally, we'll write out the config files.
 
+    # log config for the master process
+    master_log_config = generate_worker_log_config(environ, "master")
```

Comment:
Does this need the `data_dir` parameter?

Reply:
evidently. Not quite sure how I managed this.